### DISCUSSION TITLE
Main Menu "Continue"

### DISCUSSION THREAD

**Discussion by jpotelle** [EDCD/EDDI#2324] [EDDI VoiceAttack Q&A] [answered]
Is there a status variable that I can interrogate about the game state? i.e. game has loaded, game is in main "Continue" menu. I want my VA profile to know if the game is loaded or not, so I can load solo, private, open, etc. by voice. Thanks!

**Comment by Tkael** [comment, marked as answer]
The `Commander continued` event indicates the game mode that you are entering. You can store the value and use it to when swapping game modes. <https://github.com/EDCD/EDDI/wiki/Commander-continued-event>

You can also use the `Music` event to determine whether you're at the main menu. <https://github.com/EDCD/EDDI/wiki/Music-event>

**Reply by jpotelle** [reply]
Yes, I was aware of the "Commander continued" event, but it doesn't really help to detect that the event has not yet triggered at all. But I never thought to check "Music". Very helpful . Thanks!
